### DISCUSSION TITLE
表格解析方面，能否使用StructEqTable新的StructTable-InternVL2-1B模型？

### DISCUSSION THREAD

**Discussion by cpy1990** [magicpdf/Magic-PDF#778] [Ideas]
注意到StructTable-InternVL2-1B 解析latex时，可以省略掉表格本身的格式
现在版本能切换模型吗？   或者近期你们是否会更新这个模型？

**Comment by myhloli** [comment]
0.9.0已经准备封包发布了，应该是来不及适配了，如果顺利的话可能在0.9.1会适配这个
